### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by hladchenko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,48 @@
...
+- Bring includes an API for configuration to instruct the container on object creation and field injection.
+- Configuration can be stored in various formats such as XML, JSON, property text files, Java, annotations, and more.
+- All objects created by the container are stored in a context or register of objects.
+
+### Error Handling
+
+Bring gracefully handles errors with appropriate exceptions and clear error messages. Users receive instructions on how to resolve issues. Detailed logging helps diagnose problems.
+
+## License
+Bring is Open Source software released under the Apache 2.0 license.
```
Comment (DimaYakovenko) [inline README.md:48]: do we really need to state the apache licence here?

Comment (hladchenko) [reply README.md:48]: And also we already using this license. Please check https://github.com/bobocode-breskul/bring/blob/master/LICENSE

**Review by ghost — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (DimaYakovenko) [reply README.md]: okay, just clarified, whether it's not a copy paste from somewhere :)

Comment (hladchenko) [reply README.md]: > okay, just clarified, whether it's not a copy paste from somewhere :)

Could you approve the PR then? :)

**Review by OlehYakovenko — APPROVED** [review]

**Pull request merged**